### DISCUSSION TITLE
How can I grab the current driver's name?

### DISCUSSION THREAD

**Discussion by spackletoe** [alandtse/tesla#646] [Q&A]
Say I wanted to send a notification to anyone who is currently home that "Name" is headed home?

I would think it's an automation that is triggered by the destination being set to "home" and including the driver's name (through "binary_sensor.[car_name]_user_present" and some kinda look-up per the user ID? -- does this work like this?) and then appending the sensor.[car_name]_distance_to_arrival variable.

Has anyone done this one?

**Comment by StarMonkey1** [comment]
I'm looking for the exact same thing
